Makefile

```diff
@@ -13,6 +13,7 @@ deps:
 	docker-compose run --rm cli composer install
 
 depsfortests:
+	docker-compose run --rm appfortests composer update
 	docker-compose run --rm appfortests composer install
 
 depsshow:
```

Comment:
Removing `-f json` from the `composer show` command (or changing to `-f text`) [doc](https://getcomposer.org/doc/03-cli.md#show) makes for a much simpler output. We made the decision to use json, but I don't recall what the reasons were. @forevermatt do you remember?

Reply:
Here's one bit of history: https://github.com/silinternational/ssp-base/pull/61